docs/install/pxe-boot-install.md

```diff
@@ -29,9 +29,9 @@ To see sample iPXE scripts, please visit [Harvester iPXE Examples](https://githu
 Nodes need to have at least **8 GiB** of RAM because the installer loads the full ISO file into tmpfs.
 
 :::info
-The installer automatically checks if the hardware meets the [minimum requirements](https://docs.harvesterhci.io/v1.3/install/requirements/#hardware-requirements) for production use. If any of the checks fail, installation is stopped, and warnings are printed to the system console and saved to `/var/log/console.log` in the installation environment.
+The installer automatically checks if the hardware meets the [minimum requirements](./requirements/#hardware-requirements) for production use. If any of the checks fail, installation is stopped, and warnings are printed to the system console and saved to `/var/log/console.log` in the installation environment.
 
-To override this behavior, set the kernel parameter `harvester.install.skipchecks=true`. When set to `true`, warning messages are still saved to `/var/log/console.log`, but the installation proceeds even if hardware requirements for production use are not met.
+To override this behavior, set either the kernel parameter `harvester.install.skipchecks=true` or the configuration file option `install.skipchecks=true`. When set to `true`, warning messages are still saved to `/var/log/console.log`, but the installation proceeds even if hardware requirements for production use are not met.
```

Comment:
In PXE installation mode, the `harvester-configuration.yaml` should be used in most cases, as a lot of params need to be defined. From this view of point, the better choice is to set `install.skipchecks` in the configuration instead of the kernel param.  The second choice is to use kernel param.